CMakePresets.json

```diff
@@ -96,19 +102,21 @@
         "CMAKE_MAKE_PROGRAM": "$env{MAKE_PROGRAM}",
         "CMAKE_BUILD_TYPE": "Debug",
         "Kokkos_ENABLE_DEBUG_BOUNDS_CHECK": "ON",
+        "CMAKE_C_COMPILER": "cc",
         "CMAKE_CXX_COMPILER": "$env{ARTEMIS_HOME}/external/parthenon/external/Kokkos/bin/nvcc_wrapper",
         "ARTEMIS_ENABLE_CUDA": "ON",
+        "Kokkos_ARCH_HOPPER90": "ON",
         "HDF5_INCLUDE_DIR": "$env{HDF5_ROOT}/include",
         "PARTHENON_DISABLE_HDF5_COMPRESSION": "ON",
-        "Kokkos_ARCH_HOPPER90": "ON",
         "NUM_GPU_DEVICES_PER_NODE": "4"
       }
     },
     {
       "name": "venado-gpu-release",
       "cacheVariables": {
         "CMAKE_MAKE_PROGRAM": "$env{MAKE_PROGRAM}",
-        "CMAKE_BUILD_TYPE": "RelWithDebInfo",
+        "CMAKE_BUILD_TYPE": "Release",
```

Comment:
Switching it to Release makes debuggers useless for finding problems. I don't know if that's worth the shorter compile times, though.

Reply:
Yeah it's a tradeoff. My own thought is that if I hit a segfault I'm willing to rebuild in Debug mode to get symbols along with other stuff like bounds checking. But if you think always having the symbols in release-mode executables is advantageous, I'm fine with that and just let me know and I'll revert this change.

Reply:
I suppose you can just recompile, yes. But if we're talking about a "release" Release, as in it's our official release and we don't change it, then that should 100% be RelWithDebInfo. But that can be changed by hand to include the debug symbols, so I think Release is fine here.

Reply:
OK good point I have on my todo list to make a release script which I agree makes sense to be `RelWithDebInfo` because compile times/executable sizes are not really a concern in that case whereas having debug symbols is.